tests.scm

```diff
@@ -28,7 +27,7 @@
             (length (hook->list hook)))))
 
 ;; test hook-run
-(test 0 (let ((hook (make-hook 0))
+(test 1 (let ((hook (make-hook 0))
```

Comment:
The result `0` is correct, it is the test that is faulty. Both `increment` and `decrement` must be in the hook.

Can you fix it please?

Reply:
Aha! I found it weird that decrement not being used after opening the PR. But yeah it makes sense. Fixed